### ISSUE TITLE
[Suggestion] Configurable option for joined Minecarts to load chunks they are

### ISSUE THREAD

**Issue by vico93** [constellation-mc/Linkart#8]
**Is your feature request related to a problem? Please describe.**
Taking the opportunity that this mod got a rewrite and the joined minecarts ("trains") physics got a huge improvement (making that on par to TrainCarts), i decided to suggest you to add more features to allow trains to be even more useful and able to be fully automated. One of this is the ability to minecarts to run fully unattended and without the presence of a player nearby. That's why i propose this option for dynamic chunk loading.

**Describe the solution you'd like**
Basically, add an option on config to make trains only to load the chunk they are in (and the one they are heading to). That would greatly improve automated railroads building.

**Describe alternatives you've considered**
CartLoaders is a mod (abandoned i think) that allow (singular) minecarts to load chunks, but i think a integrated solution within Linkart (and only for joined minecarts) would be prefered to increase compatibility and to not make every unlinked minecart to load chunks unneeded.

**Comment by zenfyrdev** [comment]
This was surprisingly easy to add. 

There are 2 new config options: `chunkloading`, which is off by default, and `chunkloadingRadius` which is 2 by default.

A cart can load chunks only if it has a cart following it and if its velocity is not ~0. Chunkloads are persistent and should get restored after world reload.

**Comment by vico93** [comment]
Oh thank you so much!
